src/sysupdate/sysupdate.c

```diff
@@ -75,12 +75,14 @@ static Context *context_free(Context *c) {
         if (!c)
                 return NULL;
 
-        for (size_t i = 0; i < c->n_transfers; i++)
-                transfer_free(c->transfers[i]);
+        FOREACH_ARRAY(tr, c->transfers, c->n_transfers) {
+                transfer_free(*tr);
+        }
```

Comment:
no {} around single line blocks (here and everywhere)

Reply:
I mention this in other PR. Resolving to avoid duplicate comment